FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -65,6 +66,7 @@ internal FirebaseUserManager(Args args)
                     RetryOptions = args.RetryOptions,
                 });
             this.baseUrl = string.Format(IdTooklitUrl, args.ProjectId);
+            this.clock = args.Clock ?? SystemClock.Default;
```

Comment:
Did we add tests where we pass the Clock in Args?

Reply:
Yes. All tests in `FirebaseUserManagerTest`.